pset-functions.js

```diff
@@ -30,11 +30,17 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+function blackJack(playerCardScore, dealerCardScore) {
```

Comment:
@kyleligman the logic is fine but your conditionals are hard to read due to the way they are structured. See [this comment](https://github.com/jsd20190730/hw-02-functions-and-iterators/pull/4/files#r320092922) for more details.